### PR TITLE
MIXEDARCH-306: Add upgrade jobs for Azure multi-arch compute

### PR DESCRIPTION
**Pull request by Prashanth684**
Adding upgrade jobs for 4.14 and 4.15

### CODE REVIEW

**Comment by jaypoulz** [comment]
You'll probably want to `$ make update`

File: ci-operator/step-registry/openshift/upgrade/azure/heterogeneous/OWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+approvers:
```
Comment (jaypoulz) [inline ci-operator/step-registry/openshift/upgrade/azure/heterogeneous/OWNERS:1]: should we use multiarch-approvers and multiarch-reviewers for this?

**Comment by Prashanth684** [comment]
> You'll probably want to `$ make update`

already done.

File: ci-operator/step-registry/openshift/upgrade/azure/heterogeneous/OWNERS (hunk 1 above)
Comment (Prashanth684) [reply ci-operator/step-registry/openshift/upgrade/azure/heterogeneous/OWNERS:1]: yeah good point. let me change it

**Comment by Prashanth684** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-upgrade-azure-ovn-heterogeneous

**Review by jaypoulz — APPROVED** [review]
/lgtm

We still new a review from our upgrade overlords though. and good to wait for the pj-rehearsal runs.

**Comment by Prashanth684** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-upgrade-azure-ovn-heterogeneous

**Comment by Prashanth684** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-upgrade-azure-ovn-heterogeneous

**Comment by Prashanth684** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-upgrade-azure-ovn-heterogeneous

**Review by wking — APPROVED** [review]
/lgtm

**Pull request merged**